packages/tables/src/elements/Row.spec.tsx

```diff
@@ -63,8 +64,8 @@ describe('Row', () => {
     );
     const row = getByTestId('row');
 
-    fireEvent.focus(row);
-    fireEvent.blur(row);
+    userEvent.click(row);
+    row.blur();
```

Comment:
Is it ok to directly call the `blur` event?

Reply:
Yeah it's okay and encouraged:

https://testing-library.com/docs/guide-events#focusblur

https://github.com/testing-library/user-event/issues/350

Reply:
TIL